.github/workflows/release.yaml

```diff
@@ -26,15 +26,49 @@ jobs:
           role-to-assume: ${{ secrets.AWS_SAR_PUBLISHER_ROLE }}
           aws-region: us-east-2
 
+      - name: Build and Package Go executable
+        run: |
+          cd src
```

Comment:
Include 'go mod tidy' for downloading all the dependencies if any

Reply:
updated